d3a_interface/schemas.py

```diff
@@ -168,7 +172,8 @@ class ResultsSchemas:
                             "last_energy_trade_profile": {"type": "object"},
                             "last_device_statistics": {"type": "object"},
                             "last_price_energy_day": {"type": "object"},
-                            "kpi": {"type": "object"}
+                            "kpi": {"type": "object"},
+                            "area_throughput": {"type": "object"}
```

Comment:
You can enhance the validation via following [this](https://github.com/gridsingularity/d3a-interface/pull/79/files#diff-ff344600457438af0f803e1fd9897602R164-R168) approach

Reply:
Unfortunately, I can not, because the dict looks like:
```
{
  "Neighborhood 1": {
    "import": {
      "peak_energy_kWh": 0.4,
      "peak_percentage": 100.0
    }
  },
  "House 1": {
    "import": {
      "peak_energy_kWh": 0.4,
      "peak_percentage": 100.0
    }
  },
  "Neighborhood 2": {
    "export": {
      "peak_energy_kWh": 0.6,
      "peak_percentage": 200.0
    }
  },
  "House 2": {
    "export": {
      "peak_energy_kWh": 0.6,
      "peak_percentage": 200.0
    }
  }
} 
```

Reply:
okay, skip it